operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesTestSupport.java

```diff
@@ -290,12 +333,17 @@ T createResource(String namespace, T resource) {
     }
 
     Object listResources(String namespace, String fieldSelector, String... labelSelectors) {
-      throw new UnsupportedOperationException("list operation not supported");
+      if (listFactory == null)
```

Comment:
braces

Reply:
I disagree. I thought you'd read the book. Did you not get to Uncle Bob's explanation for why it is better not to use them in such cases?

Reply:
i read it cover to cover and i tried really hard to find that passage, but i was not successful. i would love to see his argument though if you can point me to it